les/api.go

```diff
@@ -52,6 +55,40 @@ func NewPrivateLightServerAPI(server *LesServer) *PrivateLightServerAPI {
 	}
 }
 
+//GatewayFee returns the current gateway fee of this light server
+func (api *PrivateLightServerAPI) GatewayFee() (gf *big.Int, err error) {
+	return api.server.handler.gatewayFee, nil
+}
+
+//SetGatewayFee allows this light server node to set a gateway fee
+func (api *PrivateLightServerAPI) SetGatewayFee(gf *big.Int) error {
+	if gf.Cmp(common.Big0) < 0 {
+		return errInvalidGatewayFee
+	}
+	if api.server.handler.gatewayFee != gf {
+		api.server.handler.gatewayFee = gf
+		if err := api.server.BroadcastGatewayFeeInfo(); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+// SetEtherbase sets the etherbase of the full node
+func (api *PrivateLightServerAPI) SetEtherbase(etherbase common.Address) error {
```

Comment:
`SetEtherbase` should be changed to `SetGatewayFeeRecipient` to match the method below and to disambiguate with https://github.com/ethereum/go-ethereum/wiki/Management-APIs#miner_setetherbase